Subject: Math::BigInt::CalcEmu documentation grammar fix
Bug-Debian: http://bugs.debian.org/443733
Origin: upstream, http://perl5.git.perl.org/perl.git/commit/384f06ae49854089e0cf13ffe34560627ea86f8a

This is fixed in blead by change 33129 but was skipped for maint-5.10.

---
 lib/Math/BigInt/CalcEmu.pm |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/lib/Math/BigInt/CalcEmu.pm b/lib/Math/BigInt/CalcEmu.pm
index 79efac6..5810f5d 100644
--- a/lib/Math/BigInt/CalcEmu.pm
+++ b/lib/Math/BigInt/CalcEmu.pm
@@ -295,7 +295,7 @@ Math::BigInt::CalcEmu - Emulate low-level math with BigInt code
 =head1 DESCRIPTION
 
 Contains routines that emulate low-level math functions in BigInt, e.g.
-optional routines the low-level math package does not provide on it's own.
+optional routines the low-level math package does not provide on its own.
 
 Will be loaded on demand and called automatically by BigInt.
 
-- 
tg: (daf8b46..) fixes/assorted_docs (depends on: upstream)
